src/forms/output/ResourcesColumns.js

```diff
@@ -0,0 +1,183 @@
+      let dataset = datasets[ dataseti ],
+          column = (
+            <ColumnSeries
+              key         = { dataset.label }
+              id          = { dataset.label.replace(` `, ``) }
+              name        = { dataset.label }
+              data        = { dataset.data }
+              legendIndex = { dataseti }
+              x           = { dataseti }
+              label       = { dataseti } />
+          );
+
+      columns.unshift(column);
+
+      if (dataset.label === `Earned`) {
+        for (let amount of dataset.data) {
+          let formatted = toFancyMoneyStr(amount);
+          categories.push(formatted);
+        }
+      }
+    }
+
+
+    const plotOptions =  { column: { stacking: `normal` }};
+    // @todo Abstract different component attributes as frosting
+    return (
+      <div className={ `benefit-columns-graph ` + (className || ``) }>
+        <HighchartsChart plotOptions={ plotOptions }>
+
+          <Chart
+            tooltip  = {{ enabled: true }}
+            zoomType = { `y` }
+            resetZoomButton = {{ theme: { zIndex: 200 }, relativeTo: `chart` }} />
+
+          <Title>{ getText(snippets.i_stackedBarGraphTitle) }</Title>
+
+          <Legend
+            align         = { `center` }
+            verticalAlign = { `top` } />
+
+          <Tooltip
+            split         = { true }
+            valuePrefix   = { `$` }
+            valueDecimals = { 2 }
+            padding       = { 8 }
+            borderRadius  = { 4 }
+            borderColor   = { `transparent`  }
+            hideDelay     = { 300 } />
+
+          <XAxis
+            endOnTick  = { false }
+            categories = { categories }
+            crosshair  = {{}}>
+
+            <XAxis.Title>{ `${timescale} ${getText(snippets.i_xAxisTitleEnd)}` }</XAxis.Title>
+            { columns }
+
+          </XAxis>
+
+          <YAxis
+            endOnTick  = { false }
+            labels     = {{ useHTML: true, formatter: this.formatMoneyWithK }}>
+
+            <YAxis.Title>{ getText(snippets.i_benefitValue) }</YAxis.Title>
+
+          </YAxis>
+
+        </HighchartsChart>
+
+      </div>
+    );
+  }  // Ends render()
+
+
+  /** Adds translation-specific money designations
+   *     (like a dollar sign for English) to the number value
+   *     string Highcharts creates, then wraps it in a span with
+   *     a class.
+   * @method
+   *
+   * @params {highchartsObject} hcObject Object Highcharts sends to event
+   *     handlers
+   *
+   * @returns {string} String representing an HTML element.
+   * 
+   * @example
+   * let toShow = formatMoneyWithk({ axis: { defaultLabelFormatter: function () { return '10k'; }} });
+   * console.log(toShow);
+   * // If app's language code is 'en':
+   * // <span class="graph-label">$10k</span>
+   * // If app's language code is 'vi':
+   * // <span class="graph-label">10k$</span>
+   */
+  formatMoneyWithK = (highchartsObject) => {
```

Comment:
I think it might be better to name this something like `formatAsHumanReadableMoneyString` (ugly, but you get the idea)--I imagine that not all localizations will use a "K".

(not blocking)

Reply:
It's not forever, but Highcharts adds a 'k' and that's why this is here. In future, we'll probably want to just do all the formatting ourselves and this won't be needed.